lib/src/lds/mrz.dart

```diff
@@ -30,41 +31,61 @@ class MRZ {
   late String _docNum;
   late String _optData;
   String? _optData2;
+  late String rawContent;
```

Comment:
Do you mean like [this](https://github.com/abdullahkaracabey/dmrtd/blob/mrz/lib/src/lds/mrz.dart) or should I create an abstract class like ElementaryFile.? I don't  want to use because ElementaryFile class has fid and sfi props

Reply:
LGTM I'd leave like that. 
Btw just nip, I think there is no need to extra reset the stream at the end of [toEncodedString](https://github.com/abdullahkaracabey/dmrtd/blob/mrz/lib/src/lds/mrz.dart#L255). Also, I'd move `toBytes` & `toEncodedString` to the top of the class after the constructor and before the `calculateCheckDigit` method, to make it more clear for the user which public methods are available.

Reply:
ok, done.